### PR TITLE
feat(github actions): adding in configuration so builds happen throug…

### PR DESCRIPTION
**Pull request by brbrown25**
…h github actions instead of travis. #50.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
-
-    scalaVersion := "3.0.0-RC1",
-
+    scalaVersion := rcVersion,
+    crossScalaVersions := crossVersions,
     useScala3doc := true,
+    skip in publish := true
   )
+
+ThisBuild / githubWorkflowJavaVersions := Seq("adopt@1.8", "adopt@11")
```
Comment (ScalaWilliam) [inline build.sbt:15]: perhaps adopt@16 too?

File: project/plugins.sbt [hunk 2]
```diff
@@ -1 +1,3 @@
 addSbtPlugin("ch.epfl.lamp" % "sbt-dotty" % "0.5.3")
+addSbtPlugin("com.codecommit" % "sbt-github-actions" % "0.10.1")
```
Comment (ScalaWilliam) [inline project/plugins.sbt:2]: this is very cool!

File: build.sbt (hunk 1 above)
Comment (brbrown25) [reply build.sbt:15]: Yeah that's a good idea. I can add that in tonight!

File: project/plugins.sbt (hunk 2 above)
Comment (brbrown25) [reply project/plugins.sbt:2]: Yeah it's so magical!

**Comment by brbrown25** [comment]
@ScalaWilliam I've pushed an update adding in jdk16. only thing that will be need is the token creation so that the actions will run

**Review by nicolasstucki — APPROVED** [review]

**Comment by brbrown25** [comment]
> @brbrown25 something failed in the CI. Do you know what happened?

I'll take a look right now!

**Comment by brbrown25** [comment]
@nicolasstucki https://github.com/scala/scala3-example-project/pull/78 this has a fully passing github workflow :)

**Pull request merged**